setup.py

```diff
@@ -46,6 +46,7 @@ def readme():
         "tqdm>=4.28.1, <5.0.0",
         "requests>=2.21.0, <3.0.0",
         "numba>=0.50.0, !=0.54.0, <0.56.0",  # Avoid 0.54 due to: https://github.com/SeldonIO/alibi/issues/466
+        "pydantic>=1.8.0, <1.9.0",
```

Comment:
Should we only upper bound at `2.0.0` or do you think it's likely to have breaking changes for minor versions?

Btw `1.9.0` is out since Dec 21 so probably should bump it either way. https://github.com/samuelcolvin/pydantic/releases